src/components/BottomNavigation.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react-native/no-inline-styles */
```

Comment:
Remove after all inline styles have been moved to the stylesheet or to a separate file